docs/user/hardware.md

```diff
@@ -74,3 +74,12 @@ _Radar(name='Prince George', institution='University of Saskatchewan', hemispher
 
 !!! Warning
     The hardware information obtained via this class contains most recent updates to the hardware file as it does not take a specific date as an input. To get specific hardware information, please use `read_hdw_file`.
+
+# Updating Radar and Hardware Information
+
+Radar hardware is infrequently updated or changed. To update your hardware files without updating or reinstalling pyDARN use the following command:
```

Comment:
Hmmm, I feel this is an awkward sentence might be just me... What do you think?

Reply:
instead of infrequently, maybe occasionally?

Reply:
```suggestion
pyDARN does not release new versions based on hardware file changes. Because hardware files change infrequently you can update the hardware files in pyDARN by using the following command:
```

Better?

Reply:
Yup!